doc/ref/matrix.xml

```diff
@@ -140,7 +140,7 @@ or the product of a finite field element and an integer matrix.
 returns the product of the row vector <A>vec</A> and the matrix <A>mat</A>.
 Probably the most usual situation is that <A>vec</A> and <A>mat</A> have the same
 lengths and are defined over a common field,
-and that all rows of <A>mat</A> have the same length <M>m</M>, say;
+and that all rows of <A>mat</A> have some common length <M>m</M>;
```

Comment:
Personally I find the phrasing with "the same" (but with "say" removed) clearer. But that may just be me. Not a blocker.

Reply:
I followed a suggestion by @wilfwilson in one of his above comments. I quite like this, so I went with it.